components/landing/SearchLanding.vue

```diff
@@ -29,8 +29,9 @@
         <a
           v-for="chain in availableChains"
           :key="chain.value"
+          v-safe-href="`/${chain.value}/explore/collectibles`"
           :class="['m-2', 'chain-option active']"
-          @click="switchChain(chain.value)">
+          @click="switchChain(chain.value, $event)">
```

Comment:
```suggestion
          @click.prevent="switchChain(chain.value)">
```
it might work with this